bundles/org.openhab.binding.ahawastecollection/README.md

```diff
@@ -95,7 +95,7 @@ actions:
 
         // Get next collection dates from items
         var biomuellDate = items['collectionDay_bioWaste'].getZonedDateTime();
-        var leichtverpackungDate = items['collectionDay_leightweightPackaging'].getZonedDateTime();
+        var leichtverpackungDate = items['collectionDay_lightweightPackaging'].getZonedDateTime();
```

Comment:
see above, this would be a documentation error because the channel is also misspelled in the binding code

Reply:
```suggestion
        var leichtverpackungDate = items['collectionDay_leightweightPackaging'].getZonedDateTime();
```